runtime/src/iree/base/tracing/tracy.h

```diff
@@ -131,6 +131,35 @@ void iree_tracing_mutex_after_lock(uint32_t lock_id);
 void iree_tracing_mutex_after_try_lock(uint32_t lock_id, bool was_acquired);
 void iree_tracing_mutex_after_unlock(uint32_t lock_id);
 
+// Entry in the Tracy file map.
+typedef struct tracy_file_contents {
```

Comment:
as with everything else in the codebase prefix with `iree_` and structs get a `_t` suffix